bindings/pydeck-carto/docs/changelog.rst

```diff
@@ -7,7 +7,7 @@ Releases and associated GitHub PRs for pydeck-carto are documented here.
 ------------
 
 0.2.0b0 - Apr 24 2024
-^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Should we mention the limitations in the changelog? What file are users most likely to consult?